test/helpers/exceptions.js

```diff
@@ -0,0 +1,44 @@
+const PREFIX = 'VM Exception while processing transaction: ';
+
+async function tryCatch(promise, message, reason) {
+  try {
+    await promise;
+    throw null;
+  } catch (error) {
+    assert(error, 'Expected an error but did not get one');
+    assert(
+    error.message.startsWith(PREFIX + message),
+    "Expected an error starting with '" +
+        PREFIX +
+        message +
+        "' but got '" +
+        error.message +
+        "' instead"
+    );
+    console.log(reason);
```

Comment:
I added the reason string because we were printing reason for every fail tx. More I use it, more I think that it's redundant. the content inside it("") is the same as this reason. Should we remove this reason and use proper describes and its?

Reply:
@satyamakgec to comment here.

Reply:
Yes, @maxsam4 we can remove the reason string. It is same as `it` statement string.